deploy/registry/registry.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    run: rig-registry
```

Comment:
Let's do just `registry` (it's still in the namespace) so we don't conflict with RIG_REGISTRY_* envs